docs/WindowsBuild.md

```diff
@@ -1,408 +1,78 @@
-# Building Swift on Windows
+# Building the Swift toolchain on Windows
 
-Visual Studio 2017 or newer is needed to build Swift on Windows. The free Community edition is sufficient to build Swift.
+The following is a step by step guide on how to build the Swift toolchain on x86/x64/ARM64 Windows.
 
-The commands below (with the exception of installing Visual Studio) must be entered in the "**x64 Native** Tools Command Prompt for VS2017" (or VS2019, VS2019 Preview depending on the Visual Studio that you are using) in the Start Menu. This sets environment variables to select the correct target platform.
+## Installing the dependencies
 
-## Install dependencies
+### Visual Studio Build Tools
 
-### Visual Studio
+Visual Studio Build Tools 2017 or newer is needed to build Swift on Windows. The free Community edition is sufficient to build the Swift toolchain. We will use the 2022 Community Edition.
```

Comment:
I don't think that this is correct - I think that the minimum might be VS2019, but even that is unclear. I would simply say that you need to use VS2022.